calculator/src/components/calculator.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import {evaluate} from "mathjs";
+import NumberFormat from 'react-number-format'
+import '../css/calculator.css'
+
+export default class Calculator extends Component {
+
+    constructor(props) {
+        super(props);
+    
+        this.state = {
+          input: ""
+        };
+      }
+
+    inputChange = (e) => {
+        this.setState({input: this.state.input + e.target.value});
+        debugger
```

Comment:
Please please please remove debuggers before committing, or at least before submitting the final version of your project.